tests/core/web3-module/test_conversions.py

```diff
@@ -18,7 +18,8 @@
         (256, b'\x01\x00'),
         (True, b'\x01'),
         (False, b'\x00'),
-    )
+    ),
+    ids=['test_' + str(x) for x in range(10)]
```

Comment:
The `ids` in this module can go too (although the trailing comma on the list of args can stay).